beta/docs/recipes/nextjs/integrate.md

```diff
@@ -1,10 +1,13 @@
 ---
 id: nextjs
-title: Integrate next js with effector
+title: Integrate Next.js with effector
+lang: en-US
 ---
 
-To do this, we will use the native [fork](https://effector.dev/docs/api/effector/fork) method.
-In the __app.tsx_ file, we create our own [Scope](https://effector.dev/docs/api/effector/scope) and wrap the application in a provider from _effector-react/scope_
+# Integrate Next.js with effector
+
+To do this, we will use the native [fork](/beta/docs/api/effector/fork.md) method.
```

Comment:
In this case you don’t need /beta/

Reply:
Fixed it. 
I just checked the availability of links through ide, but it turned out that it doesn’t work like that in beta.